docs/UserGuide.md

```diff
@@ -353,29 +384,40 @@ _Details coming soon ..._
+| **Mark Item**          | `mark [h/INDEX HOMEWORK a/INDEX ATTENDANCE]`<br> e.g., `mark a/1`                                                                                                            |
+| **Remove**             | `remove [s/INDEX SESSION h/INDEX HOMEWORK g/INDEX GRADE a/INDEX ATTENDANCE]`<br> e.g., `remove h/1`                                                                          |
+| **Show Schedule**      | `show [DDD]`<br> e.g., `show MON`                                                                                                                                            |
+| **Unmark Item**        | `unmark [h/INDEX HOMEWORK a/INDEX ATTENDANCE]`<br> e.g., `unmark h/2`                                                                                                        |
+| **Update Lesson plan** | `lesson INDEX lp/LESSON_PLAN`<br> e.g., `lesson 1 lp/Biology`                                                                                                                |
+| **View**               | `view NAME`<br> e.g., `view James Ho`                                                                                                                                        |
+
+## Prefixes summaries
+
+
+| Prefix | Definition                             | Usage                                                                                                                                                                                      | Example        |
+| -------- | ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| n/     | Name                                   | A string of alphanumeric characters. Whitespaces are allowed. Required                                                                                                                     | n/Alex Ho      |
+| p/     | Phone number                           | Numbers only. Required. Whitespaces not allowed and at least 3 digits.                                                                                                                     | p/97402341     |
+| t/     | Tag that is associated to the student. | A string of characters. Optional.                                                                                                                                                          | t/Star student |
+| s/     | Session timing of lessons              | A String of input in the format of`DDD HH:mm` <br>where DDD is three characters making up the day, HH are two numbers making up the hour and <br> mm are two numbers making up the minute. | s/tue 08:30    |
```

Comment:
does this not include the prefixes for lesson plan, grade progress, attendance and homework?

Reply:
yessss as i have said its not complete yall can add your own prefixes after the pr is merged

Reply:
HAHA nppp